cmd/trace-agent/config/config.go

```diff
@@ -326,6 +326,27 @@ func applyDatadogConfig(c *config.AgentConfig) error {
 		}
 	}
 
+	if coreconfig.Datadog.IsSet("apm_config.filter_tags_regex.require") {
+		tags := coreconfig.Datadog.GetStringSlice("apm_config.filter_tags_regex.require")
+		for _, tag := range tags {
+			splitTag := splitTagRegex(tag)
+			if containsKey(c.RequireTags, splitTag.K) {
+				continue
+			}
```

Comment:
```suggestion
			if containsKey(c.RequireTags, splitTag.K) {
			    // RequireTags already has this tag, so skip the regexp.
				continue
			}
```

Reply:
(here and below, just to make it easier for the next reader of the code 😄 )

Reply:
I added the specified comment to the code